packages/v4/src/content/design-guidelines/components/drawer/drawer.md

```diff
@@ -12,4 +12,11 @@ Drawers appear as a sliding panel that can be attached to the bottom or right ed
 
 In addition to primary-details, the drawer component is frequently used in [notification drawers](/components/notification-drawer) or terminal windows.
 
+### Splitter in a drawer
 
+A **splitter** allows you to create a layout with resizable panes. The orientation of a splitter can be set to **vertical** or **horizontal**.
```

Comment:
Can you add a link to the "splitter" component instead of bolding it?

Reply:
I'd like to, but it's not a component. It should be included in a Drawer component and it's not merged yet. :(

Reply:
It's not merged yet, but it's in the staging environment, so these guidelines will get merged when the splitter component gets merged I think!

Reply:
What if there's no splitter component?